content/content-strategy/set-up-governance/accountability-ownership/diagram.md

```diff
@@ -16,3 +16,19 @@ Set up content teams with content leads as points of escalation. Doers and speci
 With clear lines of accountability, teams are better able to work together and to break down silos.
 
 Regular reporting on measures of success is part of content governance. Reporting helps to build trust and get long-term buy-in.
+
+You can imagine this model as a pyramid that displays the hierarchy of a content team from leaders to doers and specialists:
+
+**Leaders:**
+- lead with strategy
+- champion quality
+- build capability
+- report on success
+- can say ‘no’
+
+**Doers and specialists:**
+- use content lifecycle stages
+- apply skills to tasks
+- follow guides and standards
+- design for users
+- question and check quality
```

Comment:
@sirleech has this been approved by a content designer? Say, @annekemyers or @trisha-t ? Some of these are complex terms. Do they meet the correct reading level?

Reply:
cc @joolswood

Reply:
It's been looked at, and this is the revised version. I think it will be revised more in a re-thing re: the accessible solution. Its on our kanban as well for more review

Reply:
Just read this. Alrighty then :-)